### DISCUSSION TITLE
How does Microcks generate Kafka message keys

### DISCUSSION THREAD

**Discussion by CROprogrammer** [microcks/microcks#868] [Q&A] [answered]
Hi,

Does Microcks randomly generate key for Kafka message? Is is possible to define its value through AsyncAPI schema or somehow else?

**Comment by lbroudoux** [comment, marked as answer]
Hi,

Kafka message keys are just generated as timestamps at the moment. We envision some other implementations depending on what's in the AsyncAPI spec but it still to do.
